cdap-app-fabric/src/test/java/co/cask/cdap/runtime/OpenCloseDataSetTest.java

```diff
@@ -194,4 +207,10 @@ public void testDataSetsAreClosed() throws Exception {
                         TrackingTable.getTracker("cdap.user.bar", "close"));
 
   }
+
+  @AfterClass
+  public static void tearDown() throws IOException {
+    Location location = new LocalLocationFactory(TEMP_FOLDER_SUPPLIER.get()).create(DefaultId.NAMESPACE.getId());
```

Comment:
isn't it better to construct the LocationFactory the same way, for setup and teardown?